app_allocator/tests/test_allocation_analyzer.py

```diff
@@ -60,7 +60,7 @@ def test_summarize_simple_allocation_analysis(self):
         analyzer.process_allocations_from_csv(simple_allocation_csv())
         application = analyzer.assigned[0].application
         read_counts = analyzer.analyze(analyzer.assigned)
-        summary = analyzer.summarize(read_counts)
         application_counts = read_counts[application['name']]
-        assert all([val == summary["total %s" % key]
+        summary = analyzer.summarize(analyzer.assigned)
+        assert all([val == summary[": total %s" % key]
```

Comment:
Not a blocker, but it feels a bit weird to have all the elements of summary be strings that start with ": ".

Reply:
Yeah, that's a result of the sort of hacky way the analyzer collates and reports its analysis. 
I'd love to spend time fixing that, but I don't think it's the top of the queue.